recurser/string-is
# Add a regex debugger

Adds a regular expression debugger.

<img width="1483" alt="Screen Shot 2022-01-09 at 9 18 43" src="https://user-images.githubusercontent.com/96322/148664362-975c9724-3df9-4c7c-86c9-e4241af0424c.png">

## How to test the PR

Enter a regex and a test string, and check that both single and multiple matches work.
